pkgs/development/libraries/libfive/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, eigen3_3, zlib
+, libpng, boost, qt5, guile
+}:
+
+stdenv.mkDerivation rec {
+  name = "libfive-${version}";
+  version = "755f7fed";
+
+  src = fetchFromGitHub {
+    owner  = "libfive";
+    repo   = "libfive";
+    rev    = "${version}";
+    sha256 = "1cjvmfsjbc3k5zfp22zccmjx0b2rmm2a0vm555f3bqj81yw9v7ps";
```

Comment:
Also, I get a hash mismatch, the correct one is 1ah4yw62kb2qk19ban6nv8b61fb9g8vkk5ramnp99xz3r8bjpf7c

Reply:
Weird, I must have been distracted when building this...